properties/P000068.md

```diff
@@ -14,3 +14,7 @@ A space that satisfies the selection principle
 also an open cover.
 
 See section 6 of {{doi:10.1016/0166-8641(95)00067-4}}.
+
+#### Meta-properties
+
+- This property is hereditary under closed subspaces.
```

Comment:
```suggestion
- This property is hereditary with respect to closed subspaces.
```
"hereditary with respect to" is more common in the literature (google books search for example).  It's also what Engelking is using.  So we have been using that for consistency.